mithril-signer/Dockerfile.ci

```diff
@@ -1,26 +1,29 @@
 # Creates a docker image to run an executable built outside of the image
-# This relies on the fact the mithril-signer executable has been built
+# This relies on the fact the mithril-client executable has been built
```

Comment:
@TrevorBenson could you fix this small typo?
```suggestion
# This relies on the fact the mithril-signer executable has been built
```

Reply:
Would you like a new commit with "review suggestions" in the description, or a fixup?

Reply:
A fixup is fine :+1: 
Thanks!

Reply:
Done